src/PageLayout/PageLayout.tsx

```diff
@@ -293,7 +304,7 @@ const Pane: React.FC<PageLayoutPaneProps> = ({
         merge<BetterSystemStyleObject>(
           {
             order: panePositions[computedPositionWhenNarrow],
-            display: 'flex',
+            display: isHidden ? 'none' : 'flex',
```

Comment:
Just confirming that we want `hidden` to prevent screen readers from accessing the content too.

Reply:
I believe so 👍 cc @hectahertz @alliethu @ichelsea to confirm

Reply:
If it's part of the page layout that shouldn't be able to be accessed by anyone, then we'd want to hide if from screen reader users too.